src/components/Form/ExperienceRating/index.js

```diff
@@ -0,0 +1,67 @@
+import { Component } from 'preact';
+import styles from './style';
+import { createClassName } from '../../helpers';
+import { Emoji } from 'emoji-mart';
+
+const ExperienceRatingItem = ({ value, label, description, checked, onChange }) => (
+	<div className={createClassName(styles, 'form__input-experience-rating__item')}>
+		<div className={createClassName(styles, 'form__input-experience-rating__input-container', { checked })}>
+			<input
+				type="radio"
+				name="experience-rating"
+				id={value}
+				value={value}
+				className={createClassName(styles, 'form__input-experience-rating__input')}
+				checked={checked}
+				onChange={onChange}
+			/>
+			<label className={createClassName(styles, 'form__input-experience-rating__label')} for={value}>
+				{label}
+			</label>
+		</div>
+		{checked && <span className={createClassName(styles, 'form__input-experience-rating__description')}>{description}</span>}
+	</div>
+);
+
+export class ExperienceRating extends Component {
+	state = {
+		value: this.props.value,
+	}
+
+	handleChange = (event) => {
+		const { onChange } = this.props;
+		onChange && onChange(event);
+		this.setState({ value: event.target.value });
+	}
+
+	isChecked = (itemValue) => {
+		const { value } = this.state;
+		return value === itemValue;
+	}
+
+	renderEmoji = (emojiShortName) => {
+		return Emoji({
+			set: 'apple',
+			emoji: emojiShortName,
+			size: 32,
+			sheetSize: 64,
+		  })
+	}
+
+	render() {
+		return (
+			<div
+				className={[
+					createClassName(styles, 'form__input'),
+					createClassName(styles, 'form__input-experience-rating'),
+				].join(' ')}
+			>
+				<ExperienceRatingItem value="1" label={this.renderEmoji('white_frowning_face')} description="Very bad" onChange={this.handleChange} checked={this.isChecked('1')} />
```

Comment:
You can use an Array/Map function to render `ExperienceRatingItem` from an array of objects that contains the rating items.

Reply:
Made modifications to use array maps to render the `ExperienceRatingItem` component